.github/workflows/pull_request.yml

```diff
@@ -14,6 +14,51 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  kind_setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out forklift repository
+        uses: actions/checkout@v3
+
+      - name: Checkout forkliftci
+        uses: actions/checkout@v3
+        with:
+          repository: kubev2v/forkliftci
+          path: forkliftci
+
+      - name: Bazel cache
+        id: bazel-cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/bazel
+          key: ${{ runner.os }}-bazel-cache-${{ hashFiles('WORKSPACE') }}
+
+      - name: Add cwd to path for kubectl.
+        run: echo `pwd` >> $GITHUB_PATH
+
+      - name: Get kubectl
+        run: curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+      - run: chmod u+x kubectl
+
+      - name: Build and setup everything with bazel
```

Comment:
this would be best to move to its own action so it's not so ugly here:)

Reply:
in that case a separate repo would be appropriate, not to check out but as a home for this action

Reply:
The action doesn't have a nice name, but it does most of the work, including building forklift. Not sure where else it should go and why?

Reply:
"action" - as in e.g. https://github.com/oVirt/upload-rpms-action. That would be a best place to isolate and hide the complexity of bringing up kind environment with local repo.
then you just use it here

Reply:
I see, yes, we could split it up into reusable parts.

Reply:
Or we just use https://github.com/marketplace/actions/kubernetes-kind-cluster which looks like it already does what we need.